unit-tests/unit-tests-common.h

```diff
@@ -317,64 +317,10 @@ inline bool file_exists(const std::string& filename)
 inline bool make_context(const char* id, rs2::context* ctx, std::string min_api_version = "0.0.0")
 {
     rs2::log_to_file(RS2_LOG_SEVERITY_DEBUG);
+    min_api_version;
```

Comment:
Please confirm this line is needed, and explain why if it is.

Reply:
To avoid "unused variable" warnings.

Reply:
Sorry, but I do not agree - I prefer to get a warning than seeing this - don't you?

Reply:
Normally the usual way to  do it is (that I have seen):
`(void) min_api_version;`

Reply:
You are correct, but I still do not understand exactly why... there is also `std::ignore`. I'll fix it in the dds branch when I merge there?

Reply:
ok